app/components/elements/forms/text_field_component.rb

```diff
@@ -4,6 +4,12 @@ module Elements
   module Forms
     # Component for form text fields
     class TextFieldComponent < FieldComponent
+      def initialize(label_classes: nil, **)
+        @label_classes = label_classes
```

Comment:
Just add label classes to FieldComponent? It potentially applies to all fields.

Reply:
I buy that.

Reply:
Done